ietf/templates/submit/submitter_form.html

```diff
@@ -11,12 +11,14 @@ <h3 class="mt-5">Submitter</h3>
 </p>
 {% load ietf_filters %}
 {% for author in submission.authors %}
-    <button type="button"
-           class="author btn btn-primary mb-3"
-           data-name="{{ author.name }}"
-           data-email="{% if author.email %}{{ author.email }}{% endif %}">
-        {{ author.name }}
-    </button>
+    {% if author.name %}
```

Comment:
if author.name is None for all authors in submission.authors, does the form fail?

Reply:
No, these buttons are just present to quick-fill the actual form field via javascript. If there are no buttons, then the submitter has to fill in the fields manually.